prototyping/dscp-lang-flows/hyproof-token-flows.dscp

```diff
@@ -0,0 +1,44 @@
+/*
+ * ------------------
+ * Token Declarations
+ * ------------------
+ *
+ * Note that `token`s define formats that should be persisted in transitions.
+ * Fields can be of type Role, File, Literal, a specific literal, a token type
+ * or a union of any of these
+ */
+
+token InitiatedCert {
+  hydrogen_owner: Role,
+  proposed_energy_owner: Role,
+  hydrogen_amount: Literal,
```

Comment:
need to look over the mural and make sure various pieces of information are represented here. Specifically we need a commitment to private data and maybe some other stuff